source/diagnostics-module.html

```diff
@@ -87,23 +87,30 @@ <h3>Related Resources and Modules</h3>
     Diagnostic resources for Clinical Genomic Reporting and Analysis.
     [%end-note%]
 
-    <table class="bare">
-        <tr>
-            <td><b>Name</b></td>
-            <td><b>Aliases</b></td>
-            <td><b>Description</b></td>
-        </tr>
-        [%res-item Observation%]
-        [%res-item DiagnosticReport%]
-        [%res-item ServiceRequest%]
-        [%res-item DocumentReference%]
-        [%res-item ImagingSelection%]
-        [%res-item ImagingStudy%]
-        [%res-item MolecularDefinition%]
-        [%res-item GenomicsStudy%]
-        [%res-item Specimen%]
-        [%res-item BodyStructure%]
-    </table>
+    <div>
+        <table class="bare">
+            <tr>
+                <td><b>Name</b></td>
+                <td><b>Aliases</b></td>
+                <td><b>Description</b></td>
+            </tr>
+            [%res-item Observation%]
+            [%res-item DiagnosticReport%]
+            [%res-item ServiceRequest%]
+            [%res-item DocumentReference%]
+            [%res-item ImagingSelection%]
+            [%res-item ImagingStudy%]
+            [%res-item MolecularDefinition%]
+            [%res-item GenomicStudy%]
+            [%res-item Specimen%]
+            [%res-item BodyStructure%]
+        </table>
+        <figure>Table 1: Indicates the FHIR Resources that are involved in Diagnostics.</figure>
+
+        [%impl-note%]
+        Molecular Sequence resource has been replaced by the <a href="moleculardefinition.html">Molecular Definition</a> resource. The MolecularSequence information is temporarily available for backward compatibility purposes.

```

Comment:
Hey. Kevin from CG here. If we want to keep this we can, but we should at least remove the last sentence. We have officially and completely removed MolecularSequence.